x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/hyper_parameters.tsx

```diff
@@ -398,7 +395,7 @@ export const HyperParameters: FC<Props> = ({ actions, state, advancedParamErrors
             'xpack.ml.dataframe.analytics.create.softTreeDepthToleranceText',
             {
               defaultMessage:
-                'Controls how quickly the regularized loss increases when the tree depth exceeds soft_tree_depth_limit. Must be greater than or equal to 0.01. ',
+                'Controls how quickly the loss increases when tree depths exceed soft limits. Must be greater than or equal to 0.01. ',
```

Comment:
I think the one piece of information we're missing is that smaller values mean we penalise excess depths more. How about something like:
```suggestion
                'Controls how quickly the loss increases when tree depths exceed soft limits. The smaller the value the faster the loss increases. Must be greater than or equal to 0.01. ',
```